Xsiam build (#1969)
demisto/demisto-sdk: CHANGELOG.md

* Added a new command **generate-unit-tests**.
* Fixed the destination path of the unified parsing/modeling rules in **create-content-artifacts** command.
* Fixed an issue in the **validate** command, where we validated wrongfully the existence of readme file for the *ApiModules* pack.

> * Added support to test XSIAM machines in our build, in **test-content** command.

Reviewer: ```suggestion
* Added support for running tests on XSIAM machines in the **test-content** command.
```
